helm-charts/chatqna/gaudi-values.yaml

```diff
@@ -8,6 +8,8 @@ tei:
   resources:
     limits:
       habana.ai/gaudi: 1
+  securityContext:
+    readOnlyRootFilesystem: false
```

Comment:
Instead of allowing rootfs writes, this should have been fixed by adding separate volume for writes.